clang-tools-extra/test/clang-doc/basic-project.test

```diff
@@ -383,4 +449,4 @@
 // MD-ALL-FILES: ## [GlobalNamespace](GlobalNamespace{{[\/]}}index.md)
 
 // MD-INDEX: #  C/C++ Reference
-// MD-INDEX: * Namespace: [GlobalNamespace](GlobalNamespace)
+// MD-INDEX: * Namespace: [GlobalNamespace](GlobalNamespace)
```

Comment:
Git requires a newline at the end of the file, which you've removed. Please check your editor settings to prevent that.